### PR TITLE
 Add LIP: "Update Lisk SDK modular blockchain architecture"

### PR DESCRIPTION
**Pull request by shuse2**
### Description
Proposing informational LIP to describe Lisk SDK v6 architecture and terminologies used

### CODE REVIEW

File: proposals/lip-update_lisk_sdk_modular_architecture.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
+```
+LIP: <LIP number>
+Title: Update Lisk SDK modular blockchain architecture
+Author: Shusetsu Toda <shusetsu@lightcurve.io>
```
Comment (janhack) [inline proposals/lip-update_lisk_sdk_modular_architecture.md:4]: The following line is missing:
```
Discussions-To: https://research.lisk.com/t/update-lisk-sdk-modular-blockchain-architecture/343
```

**Review by janhack — CHANGES_REQUESTED** [review]
The file header should be updated with the link to the research forum. Otherwise, it looks all good.

File: proposals/lip-update_lisk_sdk_modular_architecture.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+For example, the `seedReveal` property is added to the block asset in this hook by the [Random module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0046.md).
+
+
+## Backwards Compatibility
+
+This LIP is informational. It does not introduce any protocol change.
+
+
+## Reference Implementation
```
Comment (ricott1) [inline proposals/lip-update_lisk_sdk_modular_architecture.md:219]: Should we just drop this section?

**Review by ricott1 — APPROVED** [review]

**Review by janhack — APPROVED** [review]
Looks all good!

**Review by ishantiw — APPROVED** [review]

**Review by karmacoma — APPROVED** [review]

**Pull request merged**